package-scripts/modules/ajexport.js

```diff
@@ -0,0 +1,64 @@
+// With thanks to elenterius on discord for troubleshooting
+// https://discord.com/channels/314078526104141834/1189404550986211329/1189517519262855229
+
+/* global Project, loadModelFile, AnimatedJava */
+
+const { readdirSync, readFileSync } = require('fs');
+
+export async function script() {
+  if (typeof AnimatedJava === 'undefined') {
+    throw new Error('Failed to load Animated Java plugin before CLI plugin');
+  }
+  const paths = getConfigPaths('./package-scripts/modules/config.json');
+  const modelDir = paths.ajmodelPath.concat('/');
+  console.log('Target paths: ', paths);
+  const files = readdirSync(modelDir).filter((file) =>
```

Comment:
this doesn't search recursively yet right? can we fix that?

i found https://stackoverflow.com/a/45130990/13789724

> As of Node 20, `fs.readdir` has a `{ recursive: true }` option

so maybe

```suggestion
  const files = readdirSync(modelDir, { recursive: true }).filter((file) =>
```

it worked for me locally so 🤞

Reply:
[66260ab](https://github.com/TheAfroOfDoom/omega-flowey-minecraft-remastered/pull/64/commits/66260abeb0cf717d69ae8b104d8e2485f2958761)

Implemented and then reverted

Reply:
For now